tools/yaml-templates/build-test-publish.yml

```diff
@@ -98,6 +98,14 @@ steps:
       targetFolder: '$(Build.ArtifactStagingDirectory)\CDNFeed\$(package.version)\js'
     displayName: 'Copy TeamsJS Content for CDN'
 
+  - task: CopyFiles@2
+    inputs:
+      Contents: |
+        packages\teams-js\utils\*.json
+      TargetFolder: '$(Build.ArtifactStagingDirectory)\CDNFeed\validDomains\json'
+      flattenFolders: true
+    displayName: 'Copy validOrigins for CDN'
```

Comment:
>'Copy validOrigins for CDN [](http://example.com/codeflow?start=17&length=26)

Technically this step is copying all JSON files, not just validOrigins. As long as you are using the wildcard perhaps this description should reflect that? Or you could not use the wildcard.

Reply:
Sorry you're right, I thought about it while I was making this section and used the * to reflect if this gets renamed. I'm going to update it to just be validOrigins.json.